presto-kafka/src/main/java/io/prestosql/plugin/kafka/PlainTextKafkaProducerFactory.java

```diff
@@ -45,6 +46,7 @@ public PlainTextKafkaProducerFactory(KafkaConfig kafkaConfig)
                         .collect(joining(",")))
                 .put(ACKS_CONFIG, "all")
                 .put(LINGER_MS_CONFIG, 5)
+                .putAll(Maps.fromProperties(kafkaConfig.getSecurityConfigProperties()))
```

Comment:
We can do a static import for `fromProperties`

Reply:
will do